adapter-cqld4/pom.xml

```diff
@@ -96,6 +96,11 @@
             <artifactId>picocli</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I'm not clear on the reasons for adding these dependencies here. Is there something which affects the CVEs here?

Reply:
Correct.  These were for the medium CVEs that were addressed at the time of the PR.  It may have changed slightly, but looking at the current report, [these](https://app.snyk.io/org/nosqlbench/project/73081f87-c1b7-4c91-b407-1cb299565642) still apply at medium levels for the various modules.